.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+# Description
+
+<!-- Please describe the changes included in this PR here. -->
+
+# To-Dos
+
+<!-- Before submitting this PR, please lint and build your changes locally. -->
+<!-- Add an 'x' between the brackets to mark each checkbox as checked. -->
+
+- [ ] Lint and test with `tox`
+
+# Issues Fixed
+
+<!-- If this PR will fix/resolve an open issue on the repository, please reference it below. -->
+<!-- Otherwise, please delete this section. -->
+
+- Fixes #XXXX
```

Comment:
Please move the example into the html comment.
So nothing is rendered if lazy people forget to delete this part.

Reply:
The heading would need to be moved as well, in that case. I don't think there's a good way to handle this until GitHub supports pull request forms.

Reply:
There is also no harm in the user not deleting this section. But there is a lot of benefit to making it easy for the user to link related issues. :)

Reply:
I prefer to hint at what a good PR description could look like - I assume #XXXX will get fixed often.